src/scripts/lint/.eslintrc.react.js

```diff
@@ -1,15 +1,8 @@
 module.exports = {
-  parser: "@typescript-eslint/parser", // Specifies the ESLint parser
+  parser: '@typescript-eslint/parser',
   extends: [
-    // Uses the recommended rules from @eslint-plugin-react
-    "plugin:react/recommended",
-    // Uses the recommended rules from @typescript-eslint
-    "plugin:@typescript-eslint/recommended",
-    // Disables ESLint rules that would conflict with prettier
-    "prettier/@typescript-eslint",
-    // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors.
-    // Make sure this is always the last configuration in the extends array.
-    "plugin:prettier/recommended",
+    'eslint-config-airbnb',
+    'plugin:@typescript-eslint/recommended',
```

Comment:
Move this to a different PR and mark it as a feat that updates the listing rules for React apps.